README.md

```diff
@@ -257,6 +258,17 @@ json empty_object_explicit = json::object();
 json array_not_object = json::array({ {"currency", "USD"}, {"value", 42.99} });
 ```
 
+### Validation
```

Comment:
I do not believe validation deserves a chapter on its own. I think mentioning the existence of the accept function would be helpful somewhere where the parse methods are introduced.